browser/src/CopyNumberVariantList/CopyNumberVariants.tsx

```diff
@@ -0,0 +1,309 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react'
```

Comment:
Pretty sure I see a lot of copypasta in this file but given the circumstances I'm inclined to turn a blind eye this time. Maybe let's you and I look more carefully at this in the near future after the crunch is done.